scripts/filter

```diff
@@ -18,7 +18,7 @@ main () {
 	fi
     elif [[ "$1" =~ ${bed_pattern} ]]; then
 	echo "Sorting BED file $1" >&2
-	if ! grep -v "^#" | sort -k1,1 -k2,2n; then
+	if ! sort -k1,1 -k2,2n; then
```

Comment:
I would be a bit cautious here: since the header also has a column structure, it could get mingled among regular rows by the sort, and perplex `tabix`. It may be a coincidence that it works here. 

I would maybe consider something like:
```sh
{
  # Save stdin to temporary file
  tmp=/tmp/$$-bed-stdin 
  cat > "$tmp"
  # Output the various headers first
  grep -e '#'  -e track -e browser $tmp
  # Output the sorted data lines 
  grep -v '#' $tmp | sort -k1n; rm $tmp 
}
```

Reply:
Ah, I see what you mean! I admit that I only tested my edit with the current pop.gen. data file, and thus have scarce evidence to claim that it could be extrapolated to other BEDs we might encounter. Your suggestion looks solid to me.